source/manual/datalabs-bigquery.html.md

```diff
@@ -0,0 +1,86 @@
+---
+owner_slack: "#govuk-data-labs"
+title: Use Google BigQuery
+section: Data Labs Team Manual
+layout: manual_layout
+parent: "/manual.html"
+---
+
+
+Google BigQuery is a cloud-based SQL database. This database stores most GOV.UK visitor data. This data is primarily visitor journey data from visitors navigating through GOV.UK. BigQuery only stores visitor data if that visitor has accepted cookies.
+
+## How BigQuery data is populated
+
+Universal Analytics (UA) sends GOV.UK visitor journey data to the `govuk-bigquery-analytics.87773428.ga_sessions_intraday_YYYYMMDD` table 3 times a day, where:
+
+- `govuk-bigquery-analytics` is the project ID
+- `87773428` is the dataset name
+- `ga_sessions_intraday_YYYYMMDD` is the table name
+- `YYYYMMDD` is the current date in year-month-day format
+
+At the end of the day, UA automatically moves the data in this intraday table to a `ga_sessions_YYYYMMDD` table, and creates a new intraday table for the next day.
+
+If there are any issues with this process, you should contact a Google BigQuery administrator. The administrator will raise a request with Merkle, the agency that manages our Google Analytics / BigQuery relationship to fix the issue. Ask in the [GOV.UK Data Labs slack channel](https://gds.slack.com/archives/CHR4UQKU4) who the current admins are.
+
+The schema of these tables is defined by the [UA BigQuery Export schema](https://support.google.com/analytics/answer/3437719?hl=en). However, not all columns are filled in our tables.
+
+See the [GOV.UK Analytics page on Confluence](https://gov-uk.atlassian.net/wiki/spaces/GOVUK/pages/23855552/Analytics+on+GOV.UK#AnalyticsonGOV.UK-customDimensionsCustomdimensions) for definitions of our custom dimensions.
+
+There are many other datasets and tables on Google BigQuery. Each project usually has its own dataset.
+
+See the [example queries page](LINK) for more information.
+
+## Get access to Google BigQuery
+
+To get access to Google BigQuery, ask the Data Labs team admins to set you up with a service account and access to the Google BigQuery console. Ask in the [GOV.UK Data Labs slack channel](https://gds.slack.com/archives/CHR4UQKU4) who the current admins are.
```

Comment:
A service account is not the only way to go to authenticate, however it can be quite straight forward for new users.  We might want to rethink this recommendation at a later date, for best practice purposes. Fine for now.

Reply:
have noted on bigquery individual trello card